data-otservbr-global/scripts/actions/other/jean_pierre_food.lua

```diff
@@ -4,45 +4,45 @@ local conditions = {
 	CONDITION_FREEZING, CONDITION_DAZZLED, CONDITION_CURSED,
 	CONDITION_BLEEDING
 }
-local condition_shield = createConditionObject(CONDITION_ATTRIBUTES)
-setConditionParam(condition_shield, CONDITION_PARAM_SUBID, 3)
-setConditionParam(condition_shield, CONDITION_PARAM_BUFF_SPELL, 1)
-setConditionParam(condition_shield, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_shield, CONDITION_PARAM_SKILL_SHIELD, 10)
-setConditionParam(condition_shield, CONDITION_PARAM_FORCEUPDATE, true)
-local condition_ml = createConditionObject(CONDITION_ATTRIBUTES)
-setConditionParam(condition_ml, CONDITION_PARAM_SUBID, 4)
-setConditionParam(condition_ml, CONDITION_PARAM_BUFF_SPELL, 1)
-setConditionParam(condition_ml, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_ml, CONDITION_PARAM_STAT_MAGICPOINTS, 5)
-setConditionParam(condition_ml, CONDITION_PARAM_FORCEUPDATE, true)
-local condition_melee = createConditionObject(CONDITION_ATTRIBUTES)
-setConditionParam(condition_melee, CONDITION_PARAM_SUBID, 5)
-setConditionParam(condition_melee, CONDITION_PARAM_BUFF_SPELL, 1)
-setConditionParam(condition_melee, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_melee, CONDITION_PARAM_SKILL_MELEE, 10)
-setConditionParam(condition_melee, CONDITION_PARAM_FORCEUPDATE, true)
-local condition_dist = createConditionObject(CONDITION_ATTRIBUTES)
-setConditionParam(condition_dist, CONDITION_PARAM_SUBID, 6)
-setConditionParam(condition_dist, CONDITION_PARAM_BUFF_SPELL, 1)
-setConditionParam(condition_dist, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_dist, CONDITION_PARAM_SKILL_DISTANCE, 10)
-setConditionParam(condition_dist, CONDITION_PARAM_FORCEUPDATE, true)
-local condition_f = createConditionObject(CONDITION_ATTRIBUTES)
-setConditionParam(condition_f, CONDITION_PARAM_SUBID, 6)
-setConditionParam(condition_f, CONDITION_PARAM_BUFF_SPELL, 1)
-setConditionParam(condition_f, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_f, CONDITION_PARAM_SKILL_FISHING, 50)
-setConditionParam(condition_f, CONDITION_PARAM_FORCEUPDATE, true)
-local condition_speed = createConditionObject(CONDITION_HASTE)
-setConditionParam(condition_speed, CONDITION_PARAM_TICKS, 60 * 60 * 1000)
-setConditionParam(condition_speed, CONDITION_PARAM_SPEED, 729)
-local combat_i = createCombatObject()
-setCombatParam(combat_i, COMBAT_PARAM_EFFECT, CONST_ME_MAGIC_RED)
-setCombatParam(combat_i, COMBAT_PARAM_AGGRESSIVE, 0)
-local condition_i = createConditionObject(CONDITION_INVISIBLE)
-setConditionParam(condition_i, CONDITION_PARAM_TICKS, 600000)
-addCombatCondition(combat_i, condition_i)
+local condition_shield = Condition(CONDITION_ATTRIBUTES)
+condition_shield.setParameter(CONDITION_PARAM_SUBID, 3)
```

Comment:
Here they are not all with ":" instead of "." ? As far as I remember, the functions of the objects are accessed through the colon.